content/post/Community Unpacked.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Community Unpacked"
+slug = "alias-lucknow-journey-akshat"
+image = "images/akshat0047/i1.jpg"
+date = "2018-02-20T00:00:00"
+description = "My journey to set up a coding community in my college campus i.e. ALiAS LUCKNOW"
+disableComments = false
+author = "Akshat Pande"
+tags = ['communities','Coding','Amity','ALiAS','Open-Source']
+categories = ['niche', 'Community']
++++
+
+# COMMUNITY UNPACKED
+
+The first time, getting introduced to the world of development, I was amazed! exploring a number of technologies in the initial days of my B.Tech just for the sake of knowledge and after working on few of them at last chose, web development as my primary field.  
+One of my classmate suggested me to join [ALiAS](https://asetalias.in), a coding community of Amity University, Noida which was the main branch of our college.
+
+> ## The biggest motivation lies within **YOU**
+
+---
+
+Starting with small projects like webpages, learning **HTML + CSS + JS** like every front-end developer out there, cheers !
+
+I came across situations where I celebrated the success of my code and on the other hand irritated, when it did not worked, even after multiple attempts of mine. I asked my doubts in the community whatsapp group, became part of **great discussions, listened to experiences** etc. It was going perfect but….
+
+<br />
+
+![Team Work](/img/akshat0047/i1.jpg)
+
+Soon, I realized that there were not much people involved into such productive things in my surrounding.
+Few of them were busy in mugging up the notes and most of them in typical college activities.  
+ I came across my friends who had the will to learn, few of them even working in some fields but not progressing much.
+What we lacked was **Guidance, Healthy Discussions, Doubts, Ideas, Collaboration, Team Work** and much more but there was no appropriate solution to this in our college or surroundings.
+
+It was time and we need to set up a _COMMUNITY_ in our surrounding as well but we had no idea who will lead the same.
+
+> ## Not every time we’re in the lack of ideas but, we’re in the lack of courage to bring them alive !!
+
+---
+
+I was looking for some guidance until I came across one of my senior from the [ALiAS](https://asetalias.in) community,
+[Mr. Ayush Agarwal](https://www.thisisayush.com/) who encouraged me to open a sister community of ALiAS in Amity Lucknow.
+This marked the start of my real journey, not as a developer or mentor but as a **leader, with in my support people from various fields whether they would be my friends, head community members etc**.
+
+I showed an immense interest to transform it into a successful community which is still in the process, and the best part being, it made me learn a lot which was **not limited up to technical skills instead from management of events, to becoming a speaker, arranging webinars, promoting my juniors to indulge in the same**, overall making an experience worth sharing.
+
+<br />
```

Comment:
The rich diff's looks good, at least on GitHub. But, I am not sure how these `<br />` tags help. In markdown, if you like to break lines, just leave a line, mate. And you are good.

Reply:
I actually read it somewhere that few HTML tags works well with markdown, maybe its not a good practice. I have made the changes suggested, looking forward to see my blog published :-)

Reply:
Yes, I think almost every tag works in some form in Markdown